app/assets/stylesheets/pages/watchlist.scss

```diff
@@ -0,0 +1,80 @@
+body.page-watchlist{
+  .ribbon-container {
+		position: relative;
+	}
+
+  .ribbon{
+		position: absolute;
+		right: -5px; top: -5px;
+		z-index: 1;
+		overflow: hidden;
+		width: 75px; height: 75px;
+		text-align: right;
+    span {
+      font-size: 10px;
+      font-weight: bold;
+      color: #FFF;
+      text-transform: uppercase;
+      text-align: center;
+      line-height: 20px;
+      transform: rotate(45deg);
+      -webkit-transform: rotate(45deg);
+      width: 100px;
+      display: block;
+      box-shadow: 0 3px 10px -5px rgba(0, 0, 0, 1);
+      position: absolute;
+      top: 19px; right: -21px;
+    }
+    span::before {
+      content: "";
+      position: absolute; left: 0px; top: 100%;
+      z-index: -1;
+      border-right: 3px solid transparent;
+      border-bottom: 3px solid transparent;
+    }
+    span::after {
+      content: "";
+      position: absolute; right: 0px; top: 100%;
+      z-index: -1;
+      border-left: 3px solid transparent;
+      border-bottom: 3px solid transparent;
+    }
+  }
+
+  $ribbon-listed: #79A70A;
+  $ribbon-listed-gradient: #9BC90D;
+
+  $ribbon-ico: #1e5799;
+  $ribbon-ico-gradient: #2989d8;
```

Comment:
Change `listed` to `green`, and `ico` to `blue`. We want to keep things as generic as possible.